Tools/autotest/arducopter.py

```diff
@@ -7769,39 +7945,257 @@ def AP_Avoidance(self):
         self.context_pop()
 
     def PAUSE_CONTINUE(self):
+
+        # load auto mission
         self.load_mission("copter_mission.txt", strict=False)
 
+        # start mission
         self.set_parameter("AUTO_OPTIONS", 3)
-        self.change_mode('AUTO')
+        self.change_mode("AUTO")
```

Comment:
We use these quotes interchangeably.  Changing them from one to another may just lead to quote-type-wars and certainly increases the amount of delta to wade through.